spec/scheduling-tasks.md

```diff
@@ -264,6 +264,9 @@ see [whatwg/html#5925](https://github.com/whatwg/html/issues/5925).
   for a {{Scheduler}} |scheduler| given an {{AbortSignal}} or null |signal|, and
   a {{TaskPriority}} or null |priority|:
 
+  1. If |priority| is null, |signal| is not null and [=implements=] the {{TaskSignal}}
```

Comment:
This line uses very slightly different phrasing than the next line ("and implements" instead of "and signal implements") which makes comparing them trickier than it needs to be.